engines/wintermute/base/gfx/opengl/mesh3ds.cpp

```diff
@@ -165,4 +165,21 @@ void Mesh3DS::dumpVertexCoordinates(const char *filename) {
 	glBindBuffer(GL_ARRAY_BUFFER, 0);
 }
 
+int Mesh3DS::faceCount() {
+	// .3ds files have only triangles anyways
+	return _indexCount / 3;
+}
+
+uint16 *Mesh3DS::getFace(int index) {
+	return _indexData + 3 * index;
+}
+
+float *Mesh3DS::getVertexPosition(int index) {
+	return reinterpret_cast<float *>(_vertexData + 12 * index);
```

Comment:
Is this fully portable?

Reply:
Endianess should have been correctly handled during load, if I am not mistaken. Otherwise I don't see any issues at the moment. But then this will have to be changed anyways later, since we need space for normals and a color value.

Reply:
post this PR